rustls/src/msgs/deframer.rs

```diff
@@ -503,6 +481,21 @@ impl DeframerVecBuffer {
     fn unfilled(&mut self) -> &mut [u8] {
         &mut self.buf[self.used..]
     }
+
+    /// Append `bytes` to the end of this buffer.
+    ///
+    /// Return the span saying where it went.
```

Comment:
Should this say "range" instead of "span" to differentiate from a `FragmentSpan`?